examples/design-system/react/.gitignore

```diff
@@ -0,0 +1,24 @@
+# Logs
```

Comment:
There is already an examples/design-system/vite-react. Should these be merged or kept separate?